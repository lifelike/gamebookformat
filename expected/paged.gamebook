title = Paged Option
id = paged

= Introduction
This gamebook is formatted with the [b]paged[/b] option.
Every section should begin at the top of a new page.
Not the introsections though.

= Another Heading
This is another introsection, just to confirm that
it is not cause a new page-break.

= Formats
Only some formats support page-breaks. HTML
for instance currently does not (although it
is technically possible to add it using
print stylesheets).

* 1 start
First section. You can go to [[second]] or [[third]]
from here.

* second
This is the second section. Yay.
You can go on to [[third]] or [[end]] or back to [[start]].

* third
This is the third section. It might or might not
be numbered 3 though, because of the automatic
section numbering in gamebookformat.
You can go on to the end at [[end]].

* end
This is The End.

