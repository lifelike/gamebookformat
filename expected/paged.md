# Paged Option #


## Introduction ##
 This gamebook is formatted with the paged option. Every section should begin at the top of a new page. Not the introsections though. 

## Another Heading ##
 This is another introsection, just to confirm that it is not cause a new page-break. 

## Formats ##
 Only some formats support page-breaks (HTML, RTF, LaTeX). 

Turn to 1 to begin.
<a name="section1">

## 1 ##
 First section. You can go to [4](#section4)
 or [3](#section3)
 from here. 
<a name="section2">

## 2 ##
 This is The End. 
<a name="section3">

## 3 ##
 This is the third section. It might or might not be numbered 3 though, because of the automatic section numbering in gamebookformat. You can go on to the end at [2](#section2)
. 
<a name="section4">

## 4 ##
 This is the second section. Yay. You can go on to [3](#section3)
 or [2](#section2)
 or back to [1](#section1)
. 

