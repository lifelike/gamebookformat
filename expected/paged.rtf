{\rtf1\ansi\ansicpg1252\cocoartf1038\cocoasubrtf360
{\fonttbl\f0\fswiss\fcharset0 Helvetica;}
{\colortbl;\red255\green255\blue255;}
\paperw11900\paperh16840\margl1440\margr1440\vieww14140\viewh14860\viewkind0
\pard\tx566\tx1133\tx1700\tx2267\tx2834\tx3401\tx3968\tx4535\tx5102\tx5669\tx6236\tx6803\ql\qnatural\pardirnatural

\f0\b\fs24 \cf0

\b \qc Paged Option
\b0\
\
\b \qc Introduction
\b0\



\ql  This gamebook is formatted with the paged option. Every section should begin at the top of a new page. Not the introsections though.  \
\
\
\b \qc Another Heading
\b0\



\ql  This is another introsection, just to confirm that it is not cause a new page-break.  \
\
\
\b \qc Formats
\b0\



\ql  Only some formats support page-breaks (HTML, RTF, LaTeX).  \
\
\b Turn to 1 to begin.
\b0\
\
\page \b \qc {\*\bkmkstart s1}1{\*\bkmkend s1}
\b0\
\ql  First section. You can go to \b {\field{\*\fldinst  REF s4 \\h }{\fldrslt 4}}
\b0
 or \b {\field{\*\fldinst  REF s3 \\h }{\fldrslt 3}}
\b0
 from here.  \
\

\page \b \qc {\*\bkmkstart s2}2{\*\bkmkend s2}
\b0\
\ql  This is The End.  \
\

\page \b \qc {\*\bkmkstart s3}3{\*\bkmkend s3}
\b0\
\ql  This is the third section. It might or might not be numbered 3 though, because of the automatic section numbering in gamebookformat. You can go on to the end at \b {\field{\*\fldinst  REF s2 \\h }{\fldrslt 2}}
\b0
.  \
\

\page \b \qc {\*\bkmkstart s4}4{\*\bkmkend s4}
\b0\
\ql  This is the second section. Yay. You can go on to \b {\field{\*\fldinst  REF s3 \\h }{\fldrslt 3}}
\b0
 or \b {\field{\*\fldinst  REF s2 \\h }{\fldrslt 2}}
\b0
 or back to \b {\field{\*\fldinst  REF s1 \\h }{\fldrslt 1}}
\b0
.  \
\

}
