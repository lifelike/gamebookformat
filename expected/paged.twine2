<tw-storydata name="Paged Option" startnode="1" creator="Twine" creator-version="2.0" format="Harlowe" options="">
<style role="stylesheet" id="twine-user-stylesheet" type="text/twine-css"></style><script role="script" id="twine-user-script" type="text/twine-javascript"></script>
<tw-passagedata pid="1" name="1" tags="" position="10,10"> First section. You can go to [[4]] or [[3]] from here. </tw-passagedata>
<tw-passagedata pid="2" name="2" tags="" position="20,20"> This is The End. </tw-passagedata>
<tw-passagedata pid="3" name="3" tags="" position="30,30"> This is the third section. It might or might not be numbered 3 though, because of the automatic section numbering in gamebookformat. You can go on to the end at [[2]]. </tw-passagedata>
<tw-passagedata pid="4" name="4" tags="" position="40,40"> This is the second section. Yay. You can go on to [[3]] or [[2]] or back to [[1]]. </tw-passagedata>
</tw-storydata>
