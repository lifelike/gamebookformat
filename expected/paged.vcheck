examples/paged.json: ending (death?) section found: end
