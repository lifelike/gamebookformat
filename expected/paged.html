<!DOCTYPE html>
<html>
  <head>
    <meta charset='utf-8'>
    <meta name="viewport"
        content="width=device-width, initial-scale=1, maximum-scale=1">
    <title>Paged Option</title>
    <script type="text/javascript" src="gamebookformatplay.js"></script>
    <link rel="stylesheet" href="gamebookformat.css"
          type="text/css" />
    <script>
      if (typeof gamebook !== 'undefined') {
         gamebook.id = 'paged';
      }
    </script>
  </head>
  <body>
<div class="hideintrolink nodisplay"
    onclick="gamebook.hideIntroSections()">(hide instructions)</div>
  <div class="gamebook">
<div class="introsection">
<div class="introsectionheading">Introduction</div>
<div class="introsectionbody">
 This gamebook is formatted with the <b>paged</b>
 option. Every section should begin at the top of a new page. Not the introsections though. 
</div>
</div>
<div class="introsection">
<div class="introsectionheading">Another Heading</div>
<div class="introsectionbody">
 This is another introsection, just to confirm that it is not cause a new page-break. 
</div>
</div>
<div class="introsection">
<div class="introsectionheading">Formats</div>
<div class="introsectionbody">
 Only some formats support page-breaks. HTML for instance currently does not (although it is technically possible to add it using print stylesheets). 
</div>
</div>
<div class="resumelink nodisplay"
  onclick="gamebook.loadGame()">Resume saved game.</div>

<div class="startlink"
  onclick="gamebook.turnTo(1)">Turn to 1 to begin.</div>
<script>
    if (typeof gamebook !== 'undefined' && gamebook.hasSavedGame()) {
        var resumeLinks = document.getElementsByClassName('resumelink');
        Array.prototype.forEach.call(resumeLinks, function(e) {
            e.classList.remove('nodisplay');
        });
    }
</script>
<div class="section" id="section1">
<div class="sectionnumber" id="para1">1</div>
<div class="sectiontext">
 First section. You can go to <a class="sectionref enabledlink" data-ref="4"
href="#section4">4</a> or <a class="sectionref enabledlink" data-ref="3"
href="#section3">3</a> from here. 
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(1, document.getElementById('section1'));
}
</script><div class="section" id="section2">
<div class="sectionnumber" id="para2">2</div>
<div class="sectiontext">
 This is The End. 
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(2, document.getElementById('section2'));
}
</script><div class="section" id="section3">
<div class="sectionnumber" id="para3">3</div>
<div class="sectiontext">
 This is the third section. It might or might not be numbered 3 though, because of the automatic section numbering in gamebookformat. You can go on to the end at <a class="sectionref enabledlink" data-ref="2"
href="#section2">2</a>. 
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(3, document.getElementById('section3'));
}
</script><div class="section" id="section4">
<div class="sectionnumber" id="para4">4</div>
<div class="sectiontext">
 This is the second section. Yay. You can go on to <a class="sectionref enabledlink" data-ref="3"
href="#section3">3</a> or <a class="sectionref enabledlink" data-ref="2"
href="#section2">2</a> or back to <a class="sectionref enabledlink" data-ref="1"
href="#section1">1</a>. 
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(4, document.getElementById('section4'));
}
</script>    <div id="counters" class="counters">
    </div>
    <div id="counterTemplate" class="counterTemplate">
      <span class="counterheading"></span>
      <span class="countercontents"></span>
    </div>
    </div>
    <div id="collections" class="collections">
    </div>
    <div id="collectionTemplate" class="collectionTemplate">
      <span class="collectionheading"></span>
      <span class="collectioncontents"></span>
    </div>
    </div>
    <script>
if (typeof gamebook !== 'undefined') {
   gamebook.prepare();
}
    </script>
<div class="displayintrolink nodisplay"
    onclick="gamebook.showIntroSections()">(show instructions)</div>
    </div>
  </body>
</html>
